Changelog.md

```diff
@@ -2,7 +2,69 @@
 
 ### Gazebo Rendering 10.X
 
-### Gazebo Rendering 10.0.0 (20XX-XX-XX)
+### Gazebo Rendering 10.0.0 (2025-09-XX)
+
+1. **Baseline:** this includes all changes from 9.3.0 and earlier.
+
+1. CppCheck fixes
+    * [Pull request #1148](https://github.com/gazebosim/gz-rendering/pull/1148)
```

Comment:
In 9.3.0

Reply:
removed. [3712675](https://github.com/gazebosim/gz-rendering/pull/1158/commits/37126756b96db6f57df30ffe7fd5c2189a644a44)